.env.sample

```diff
@@ -26,3 +26,5 @@ EXPERIMENTS_PAGINATE_BY=50
 
 # Enables the mozlog JSON format for console logging.
 LOGGING_USE_JSON=True
+# This will print all sent emails to stdout without requiring a server.
+EMAIL_BACKEND=django.core.mail.backends.console.EmailBackend
```

Comment:
The devs working on experimenter, if this PR is accepted, will need to update their personal `.env` files to have this change. Otherwise the default is going to be to try to actually send emails over SMTP.